disposable_email_blacklist.conf

```diff
@@ -1544,6 +1560,7 @@ secretemail.de
 secure-mail.biz
 secure-mail.cc
 secured-link.net
+securehost.com.es
```

Comment:
Interesting, the securehost.com.es website doesn't work now (Connection refused).